packages/shared/src/utils/contract/contract-types.ts

```diff
@@ -0,0 +1,80 @@
+import { ContractUpdate } from '@deriv/api-types';
+
+export type TGetFinalPrice = {
+    sell_price: number;
+    bid_price: number;
+};
+
+export type TIsValidToSell = TIsEnded & {
+    is_valid_to_sell: 0 | 1;
+};
+
+export type TGetTotalProfit = {
+    bid_price: number;
+    buy_price: number;
+};
+
+export type TGetDisplayStatus = TGetTotalProfit & {
+    status: 'open' | 'sold' | Omit<string, 'open' | 'sold'>;
+};
+
+export type TTickStream = {
```

Comment:
As for me 'Stream' sounds like something big, what includes some small parts.
I mean, maybe more proper name for one tick is, let say, TTickItem, or just TTick.
and stream will be TTick[]

Reply:
thank u for your revision, I'll change this